apps/dc_tools/odc/apps/dc_tools/_stac.py

```diff
@@ -404,3 +413,21 @@ def stac_transform(input_stac: Document, relative: bool = True) -> Document:
 def transform_geom_json_coordinates_to_list(geom_json):
     geom_json["coordinates"] = numpy.array(geom_json["coordinates"]).tolist()
     return geom_json
+
+
+def ds_to_stac(ds: Dataset) -> dict:
+    """Get STAC document from dataset with eo3 metadata"""
+    if ds.is_eo3:
+        if not ds.uris:
+            raise ValueError("Can't find dataset location")
+        location = ds.uris[0]
+        stac = to_stac_item(
+            from_doc(ds.metadata_doc, skip_validation=True),
+            location,
+            explorer_base_url="https://explorer.dea.ga.gov.au",
```

Comment:
If this is optional, we should leave it out instead of defaulting to DEA. We're not the only ones to use these tools.

Reply:
Removed